### DISCUSSION TITLE
USDZ models are too dark in AR Quick look

### DISCUSSION THREAD

**Discussion by winghang4** [google/model-viewer#3529] [Q&A]
The model is converted fine and looks ok in Quick Look on my mac but when I look at it in Quick Look on my iPhone the model is too dark. 
Does anyone have methods to solve this problem in model-viewer HTML code?

**Comment by elalish** [comment]
That would be because of AR lighting estimation. Is it any better in a differently-lit room? @grorg on the Apple side.

**Comment by futahei** [comment]
Is the problem something that is happening before iOS 16?
I had a similar problem, but it improved when I updated to iOS16.
Please see the related thread attached.

[QuickLook environment probes are always much too dark](https://developer.apple.com/forums/thread/654518)
[USDZ model is too dark in AR Quick look](https://developer.apple.com/forums/thread/116534)
